packages/core/src/components/TermList/Term.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import withStyles, { WithStylesProps } from '../../composers/withStyles';
+
+import Row from '../Row';
+import Spacing from '../Spacing';
+import Text from '../Text';
+
+function Term({
+  label,
+  after,
+  endAlign,
+  keyTextRegular,
+  children,
+  cx,
+  styles,
+}: {
+  label: string;
+  after?: NonNullable<React.ReactNode>;
+  endAlign?: boolean;
+  inline?: boolean;
+  keyTextRegular?: boolean;
```

Comment:
let's make regular the default and small the optional? that'd be more inline with our sizing prop patterns

Reply:
Agreed that `keyTextRegular` is a super weird prop name, I'm changing it to `small` with the appropriate functionality, but setting `small = true` by default since that's how DataItem is used in pretty much every use case I"m aware of